### ISSUE TITLE
[CRASH] OpenSIPS 2.4.6 crash

### ISSUE THREAD

**Issue by vtzan** [OpenSIPS/opensips#1857]
Dear Team,

I have just upgraded one of my production opensips instances to 2.4.6 and processes segfaults every x minutes. Here are the kernel logs and below are the core dump bt full.

I am using create_dialog("rR") in my script. I don't know if this is a problem....

```
[Sat Oct 12 21:41:18 2019] opensips[10598]: segfault at 0 ip           (null) sp 00007ffcad420658 error 14 in opensips[400000+2c6000]
[Sat Oct 12 21:46:24 2019] opensips[11233]: segfault at 0 ip           (null) sp 00007fffd2268fe8 error 14 in opensips[400000+2c6000]
[Sat Oct 12 21:51:29 2019] opensips[11856]: segfault at 0 ip           (null) sp 00007fff1a706eb8 error 14 in opensips[400000+2c6000]
[Sat Oct 12 21:56:35 2019] opensips[12517]: segfault at 0 ip           (null) sp 00007ffe74b07ca8 error 14 in opensips[400000+2c6000]
[Sat Oct 12 22:01:40 2019] opensips[13235]: segfault at 0 ip           (null) sp 00007fff7fa08cc8 error 14 in opensips[400000+2c6000]
[Sat Oct 12 22:06:50 2019] opensips[13978]: segfault at 0 ip           (null) sp 00007fff8e947978 error 14 in opensips[400000+2c6000]
```
**OpenSIPS version you are running**

```
version: opensips 2.4.6 (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, F_MALLOC, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: edef893c5
main.c compiled on 19:46:33 Oct 12 2019 with gcc 5.4.0

```

**Crash Core Dump**
Crash BT FULL ---> https://pastebin.com/HWv4EVL1

**Relevant System Logs**
Oct 12 21:41:19 sbc1 /usr/sbin/opensips[10598]: CRITICAL:core:sig_usr: segfault in process pid: 10598, id: 14
Oct 12 21:46:24 sbc1 /usr/sbin/opensips[11233]: CRITICAL:core:sig_usr: segfault in process pid: 11233, id: 17
Oct 12 21:51:30 sbc1 /usr/sbin/opensips[11856]: CRITICAL:core:sig_usr: segfault in process pid: 11856, id: 18
Oct 12 21:56:35 sbc1 /usr/sbin/opensips[12517]: CRITICAL:core:sig_usr: segfault in process pid: 12517, id: 19
Oct 12 22:01:41 sbc1 /usr/sbin/opensips[13235]: CRITICAL:core:sig_usr: segfault in process pid: 13235, id: 9

**OS/environment information**
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial

Thank you in advance for your support
Vasilios Tzanoudakis

**Additional context**

**Comment by liviuchircu** [comment]
Hi, @vtzan !  If we look closely at the commits:

```commit 072b7a678636c2a2bb90ad8bd567a6c610feeaf2
Author: Liviu Chircu <liviu@opensips.org>
Date:   Wed May 22 12:48:52 2019 +0300

    dialog: Do not replicate CSeq changes w/o a cluster
    
    (cherry picked from commit 65ae04c892d2869be4a56049adcea70666e97dc8)

commit edef893c5af7c8b9dfc516e79ef78ba69d5b3651 (tag: 2.4.6)
Author: Razvan Crainea <razvan@opensips.org>
Date:   Tue Jun 11 17:56:54 2019 +0300

    Update ChangeLog for 2.4.6

```

The fix for your bug was included in the `2.4` tree right after the commit that you're running :) There have been lots of critical fixes in `2.4` within the last months, so I highly recommend running the latest code if you're on `2.4`, until we push a new tag to include all the recent work.

**Comment by vtzan** [comment]
Dear @liviuchircu ,

Thank you for your fast reply.
I have compiled the latest 2.4 code as soon as I faced this issue and until now I am still monitoring and I haven't faced that issue again so yes that was the issue.

thank you once again for the fast reply on this.

Best Regards
Vasilios Tzanoudakis
